lib/model/folder.go

```diff
@@ -256,7 +256,7 @@ func (f *folder) Delay(next time.Duration) {
 
 func (f *folder) Stop() {
 	f.cancel()
-	<-f.stopped
+	f.Service.Stop()
```

Comment:
Shouldn't we just do defer f.cancel() in serve?

Reply:
The cancelled context is also used in functions within serve, e.g. puller, so that would be too late.

Reply:
I guess we could in the run do
```
go func()
  select {
   case <-stop:
     f.cancel()
  }
}()
```

Yet I don't feel strongly about this.